### ISSUE TITLE
Failed to execute script m64py

### ISSUE THREAD

**Issue by Papermanzero** [mupen64plus/mupen64plus-ui-python#107]
M64Py cannot be started. but instead it shows a warning message "Failed to execute script m64py"
The issue is caused in Version 0.2.4 with a "new" audio plugin.
It can be downloaded in this issue:
https://github.com/gonetz/GLideN64/issues/470#issuecomment-299810150

Also Version 0.2.5 shows the message, independet of old or new plugins.

Operating System: Windows 10 - Creators Update.

**Comment by loganmc10** [comment]
It's probably because that audio plugin I posted is a 64-bit version and you've got a 32-bit version of m64py/mupen64plus.

Here is a 32-bit version of that "new" audio plugin:
https://drive.google.com/open?id=0B8iAuv3fmhb_SlRsMTl3WFVoNkE

**Comment by Papermanzero** [comment]
Thanks for the 32-bit Version. But it still has the issue.
Warning message is the same with 0.2.4.

**Comment by gen2brain** [comment]
These builds are with console window, start it from cmd so hopefully you can see the real error:
http://81.4.106.254/m64py-0.2.5-setup-console.exe
http://81.4.106.254/m64py-0.2.5-portable-console.zip

**Comment by Papermanzero** [comment]
The cmd gives following traceback:

Traceback (most recent call last):
  File "m64py", line 58, in <module>
  File "m64py\frontend\mainwindow.py", line 81, in __init__
  File "m64py\frontend\settings.py", line 47, in __init__
  File "m64py\frontend\settings.py", line 69, in add_items
  File "m64py\frontend\input.py", line 52, in __init__
  File "m64py\frontend\input.py", line 126, in add_items
AttributeError: 'str' object has no attribute 'decode'
Failed to execute script m64py

**Comment by Papermanzero** [comment]
@gen2brain 
I hope this info is sufficient. If you need more support i can maybe help

**Comment by gen2brain** [comment]
This should be fixed in c3a037f7b66842c516b7a223976bb05f91946379 .
New Windows builds are here:
http://81.4.106.254/m64py-0.2.5-setup.exe
http://81.4.106.254/m64py-0.2.5-portable.zip
http://81.4.106.254/m64py-0.2.5-setup-console.exe
http://81.4.106.254/m64py-0.2.5-portable-console.zip

**Comment by Papermanzero** [comment]
@loganmc10  and @gen2brain 
The new version is working without any issues.
Except if I am using the optimised audio plugin.

With the "standard" version I get the same error as before:
Failed to execute script m64py
With the console version I have to select the core lib and afterwards the emu crashes.

```
Frontend: ERROR: 'Mupen64Plus Core' library not found.
Traceback (most recent call last):
  File "m64py\core\core.py", line 92, in core_load
Exception: 'Mupen64Plus Core' library not found.
Frontend: INFO: attached to library 'Mupen64Plus Core' version 2.5.0
Frontend: INFO: includes support for Dynamic Recompiler.
Frontend: INFO: video extension enabled
Unhandled Python exception
```

**Comment by Papermanzero** [comment]
I managed to get more information, if I open the application directly.

Traceback (most recent call last):
File "m64py\core\core.py", line 214, in plugin_load_try
File "ctypes\__init__.py", line 425, in LoadLibrary
File "ctypes\__init__.py", line 347, in __init__
OSError: [WinError 126] The used modul was not found (-> translated from german)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "m64py", line 58, in <module>
File "m64py\frontend\mainwindow.py", line 90, in __init__
File "m64py\frontend\worker.py", line 55 in init
File "m64py\frontend\worker.py", line 138 in plugins_load
File "m64py\core\core.py", line 223 in plugin_load_try
AttributeError: 'str' object has no attribute 'decode'
Failed to execute script m64py

**Comment by gen2brain** [comment]
Should be fixed in #0331188 , but that is just to print path of plugin that failed to load.
New builds are here:
http://81.4.106.254/m64py-0.2.5-setup-console.exe
http://81.4.106.254/m64py-0.2.5-portable-console.zip

I also tried to start it with https://github.com/loganmc10/mupen64plus-audio-sdl win32 binary build that is linked, it is not a static build so I needed libgcc_s_dw2-1.dll , libsamplerate-0.dll and libspeexdsp-1.dll , but it crashes anyway at startup.

I tried also on Linux and it works there.
